testdata/scripts/literals.txt

```diff
@@ -74,20 +74,6 @@ const (
 Second Line`
 )
 
-const (
-	i       = 1
-	boolean = true
-
-	skip1 = "Skip this block"
-)
-
-const (
-	foo = iota
-	bar
-
-	skip2 = "also skip this"
-)
-
```

Comment:
why delete these lines? at least we can check that we're not breaking iotas, for instance, which was a bug we had a while ago.

I think I'd leave the test script untouched, assuming nothing changes in terms of its behavior.

Reply:
I don't move constants to variables anymore, therefore it only makes sense to test const once which is done above.

Reply:
I still don't think I agree - this bit of code tests other edge cases like iota and non-string constants. Even if the new code doesn't touch those right now, they still exercise edge cases that might break again in the future. iota was a bug we had to fix, for instance.